TimerPao/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>SuperMídia Widget</title>
+	<meta charset="UTF-8">
+	<meta content="SuperMídia Widget" property="og:title">
+	<meta name="author" content="">
```

Comment:
Pode botar os creditos 😜

```suggestion
	<meta name="author" content="Patricia Conceição">
```